tests/api/test_remote_engine.py

```diff
@@ -35,16 +35,17 @@ class MockServer:
     def __init__(self):
         self.request_count = 0
 
-    def get_job_status(self, _id):
+    def get_job(self, _id):
         """Returns a 'queued' job status until the number of requests exceeds a defined
         threshold, beyond which a 'complete' job status is returned.
         """
         self.request_count += 1
-        return (
+        status = (
             JobStatus.COMPLETED
             if self.request_count >= self.REQUESTS_BEFORE_COMPLETED
             else JobStatus.QUEUED
         )
+        return Job(id_="123", status=status, connection=None)
```

Comment:
A small dummy meta dictionary could be added here, and then later checked similarly to the `status`.

Reply:
Good idea, this has been added!